node/chainSimulator/components/processComponents.go

```diff
@@ -155,12 +154,25 @@ func CreateProcessComponents(args ArgsProcessComponentsHolder) (*processComponen
 		return nil, err
 	}
 
-	whiteListRequest, err := disabled.NewDisabledWhiteListDataVerifier()
+	lruCache1, err := cache.NewLRUCache(100000)
```

Comment:
can use a const and also rename lruCache1 and lruCache2 to something more relevant

Reply:
refactored